src/core/diags/factory_diags.cpp

```diff
@@ -188,6 +188,7 @@ const struct Diags::Command Diags::sCommands[] = {
     {"stats", &Diags::ProcessStats},
     {"stop", &Diags::ProcessStop},
     {"stream", &Diags::ProcessStream},
+    {"sweep", &Diags::ProcessSweep},
```

Comment:
Could you update the `README.md` to describe how to use this new diag command?

Reply:
Done